internal/cmd/ske/kubeconfig/create/create.go

```diff
@@ -50,8 +53,8 @@ func NewCmd() *cobra.Command {
 				`Create a kubeconfig for the SKE cluster with name "my-cluster" and set the expiration time to 2 months`,
 				"$ stackit ske kubeconfig create my-cluster --expiration 2M"),
 			examples.NewExample(
-				`Create a kubeconfig for the SKE cluster with name "my-cluster" in a custom location`,
-				"$ stackit ske kubeconfig create my-cluster --location /path/to/config"),
+				`Create a kubeconfig for the SKE cluster with name "my-cluster" in a custom filepath`,
+				"$ stackit ske kubeconfig create my-cluster --filepath /path/to/config"),
```

Comment:
Having the file extension would make this example much more clear, or does the kubeconfig not have any?